src/components/impagination.js

```diff
@@ -86,74 +154,15 @@ export default class Impagination extends Component {
 
   // update state on mount to trigger any initial requests and
   // load any already-resolved records
-  componentWillMount() {
-    this.updateDataset();
-  }
+  // componentWillMount() {
+  //   this.updateDataset();
+  // }
 
   // the main trigger for updating the datset state
-  componentWillReceiveProps(nextProps) {
-    this.updateDataset(nextProps);
-  }
-
-  //  Updates the dataset immutably and triggers the `fetch` property
-  //  when there are unrequested pages.
-  updateDataset(props = this.props) {
-    let { collection, pageSize, loadHorizon, readOffset, fetch } = props;
-    let dataset = this.state.dataset;
+  // componentWillReceiveProps(nextProps) {
```

Comment:
remove comments

Reply:
That was sloppy. Thanks for the keen eye